ckanext/canada/templates/internal/snippets/user.html

```diff
@@ -32,7 +32,7 @@ <h2 class="wb-inv">Sign-on information</h2>
 
             {% block header_get_help %}
             <li>
-                <a href="mailto:open-ouvert@tbs-sct.gc.ca?subject={{ _('Help using the Open Government Registry') }}"
+                <a href="{% if h.lang() == 'fr' %}/fr/guide-operations-registre/ton-compte{% else %}/en/registry-operations-guide/your-account{% endif %}"
```

Comment:
I removed the domain so that it can redirect to staging/test if not on prod environment which is useful during testing

Reply:
is the redirect in our registry nginx config?

We have a `h.portal_url()` helper that gets the portal URL from the ckan ini file so it can point to the right place for the given site (prod, staging, test)